apps/bot/commands/slash/get-answer-count.ts

```diff
@@ -0,0 +1,41 @@
+import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommand } from '../types.js'
+import {
+  getCorrectAnswersCount,
+  isUserProfilePublic,
+} from '../../db/actions/users.js'
+import { env } from '../../env.js'
+export const command: SlashCommand = {
+  data: new SlashCommandBuilder()
+    .setName('get-answer-count')
+    .setDescription('Get the number of your accepted answers'),
```

Comment:
As this command needs guild context, you should disable it in DMs with `.setDMPermission(false)`